### PR TITLE
Update to `camunda-bpmn-js@1.1.1`

### PR DESCRIPTION
**Pull request by nikku**
#### New changes

* `FIX`: serialize templated properties in stable order ([bpmn-js-properties-panel#838](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/838))
* `FIX`: do not sort IO mappings alphabetically ([bpmn-js-properties-panel#845](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/845), [bpmn-js-properties-panel#843](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/843))
* `FIX`: (new/regression) correctly update error message (https://github.com/camunda/camunda-modeler/issues/3392)

See also [CHANGELOG](https://github.com/camunda/camunda-bpmn-js/blob/main/CHANGELOG.md#110).

Closes https://github.com/camunda/camunda-modeler/issues/3392.

### CODE REVIEW

**Comment by nikku** [comment]
Odd issue discovered (works on latest 5.7.0 release):

When updating an error `correlationKey` then the error ("required") is no set or cleared correctly:

capture 3UixOB_optimized

**Comment by nikku** [comment]
Spun that bug off into a separate bug issue: https://github.com/camunda/camunda-modeler/issues/3392.

**Comment by marstamm** [comment]
Do we want to create a separate PR for https://github.com/camunda/camunda-modeler/issues/3392 or bump to the patch versions in this one?

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,15 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
-### BPMN
+* `DEPS`: update to `camunda-bpmn-js@1.1.1`
```
Comment (marstamm) [inline CHANGELOG.md:9]:
```suggestion
* `DEPS`: update to `camunda-bpmn-js@1.1.2`
```

Comment (nikku) [reply CHANGELOG.md:9]: These patch bumps...

**Pull request merged**